docs/docs/policies.mdx

```diff
@@ -87,18 +87,25 @@ The Transformer Embedding Dialogue (TED) Policy is described in
 This policy has a pre-defined architecture, which comprises the
 following steps:
```

Comment:
I think I'm missing some high-level overview of the different things that TED does. Could we add such thing before enumerating these steps? From the detailed steps below I get the idea that TED 1) predicts next action based on the starspace-inspired similarity (though this isn't said explicitly), 2) predicts entities (in the most recent user turn?), but it requires a lot of effort to extract this high-level info from the steps.

Reply:
copied description form DIET